floss/main.py

```diff
@@ -546,28 +546,38 @@ def main(argv=None) -> int:
 
     static_runtime = get_runtime_diff(interim)
 
-    lang_id = identify_language(sample, static_strings)
-
     # set language configurations
-    if (lang_id == Language.GO and args.language == "") or args.language == Language.GO.value:
+    lang_id: Language
+    if args.language == Language.GO.value:
+        lang_id = Language.GO
+    elif args.language == Language.RUST.value:
+        lang_id = Language.RUST
+    elif args.language == Language.DOTNET.value:
+        lang_id = Language.DOTNET
+    elif args.language == "none":
+        lang_id = Language.UNKNOWN
+    else:
+        lang_id = identify_language(sample, static_strings)
```

Comment:
```suggestion
    lang_id_mapping = {
        Language.GO.value: Language.GO,
        Language.RUST.value: Language.RUST,
        Language.DOTNET.value: Language.DOTNET,
        "none": Language.UNKNOWN,
    }

    lang_id = lang_id_mapping.get(args.language, identify_language(sample, static_strings))
```
I suggest adopting this mapping approach for language identification. What do you think?

Reply:
that's neat, please submit a PR as this is already merged now

Reply:
Okay! 😄